lib/tasks/backfill_applicants.rake

```diff
@@ -0,0 +1,88 @@
+require 'csv'
+require 'debug'
+require "net/http"
+
+def grab_location(location_string)
+  return if location_string.nil?
+  uri = URI("https://nominatim.openstreetmap.org/search?q=\"#{location_string}\"&format=json&&addressdetails=1&accept-language=en")
+  response = Net::HTTP.get_response(uri)
+  location_data = JSON.parse(response.body)[0]["address"]
+  
+  [location_data["city"], location_data["state"], location_data["country"]]
+end
+
+task backfill_applicant_responses: :environment do
+  headers = ["#","name","email","reason_for_applying","linkedin_url","learned_to_code","location","project_experience","available_hours_per_week","referral_source","additional_information","start_utc","submit_utc","network_id","tags"]
+  table = CSV.read(Rails.root.join('lib', 'tasks', 'responses.csv'), headers:,  liberal_parsing: true)
```

Comment:
Assuming your plan is to:
1. ssh in
2. rails c
3. create a file with this name and copy the data into it (via console)
4. consume

Reply:
Alternatively what you can do is run this locally and export the data (in the table) as json (well link it via email or something). This way we only need to create account and then jsut copy/paste json into variable and practically consume. Your call though in how you'd want to do this just throwing out another idea.

Reply:
yeah, I figured that I'd tackle that implementation detail once I got others' eyes on the core task that works, especially b/c there are some data integrity issues that I figure we should all be aware of. Fundamentally, though, it's a small modification, assuming the rest of the script works (which it does, locally, w/ the cleaned datasets).